commands/updateListUsers.js

```diff
@@ -0,0 +1,71 @@
+/* eslint no-await-in-loop: "off", no-restricted-syntax: "off", no-console: "off" */
+/* eslint func-names: "off" */
+/**
+ * @module updateListUsers
+ * @description Updates all checkins for all users to make sure the attributes
+ * are updated with their respective lists.
+ */
+
+const mongoose = require('mongoose');
+const app = require('../');
+
+const store = app.config.env[process.env.NODE_ENV].database.stores[process.env.NODE_ENV];
+mongoose.connect(store.uri, store.options);
+
+const User = require('../api/models/User');
+
+async function run() {
+  const childAttributes = User.listAttributes();
+  const cursor = User
+    .find({})
+    .populate([{ path: 'lists.list' },
+      { path: 'operations.list' },
+      { path: 'bundles.list' },
+      { path: 'disasters.list' },
+      { path: 'organization.list' },
+      { path: 'organizations.list' },
+      { path: 'functional_roles.list' },
+      { path: 'offices.list' },
+    ])
+    .cursor();
+
+  /* eslint no-await-in-loop: "off" */
+  for (let user = await cursor.next(); user != null; user = await cursor.next()) {
+    for (let i = 0; i < childAttributes.length; i += 1) {
+      const childAttribute = childAttributes[i];
+      let lu = {};
+      if (childAttribute === 'organization') {
+        lu = user[childAttribute];
+        if (lu && lu.list && lu.list.owner) {
+          lu.owner = lu.list.owner;
+          lu.managers = lu.list.managers;
+          // logger.info(`Updated list for ${user._id.toString()}`);
+          /* eslint no-await-in-loop: "off" */
+          await user.save();
+        } // else {
+        // logger.info(`No list for ${user._id.toString()}`);
+        // }
+      } else {
+        for (let j = 0; j < user[childAttribute].length; j += 1) {
+          if (lu && lu.list && lu.list.owner) {
+            lu.owner = lu.list.owner;
+            lu.managers = lu.list.managers;
+            // logger.info(`Updated list for ${user._id.toString()}`);
+            /* eslint no-await-in-loop: "off" */
+            await user.save();
+          } // else {
+          // logger.info(`No list for ${user._id.toString()}`);
+          // }
+        }
+      }
+    }
+  }
+  process.exit();
+}
+
+(async function () {
+  await run();
+}()).catch((e) => {
+  console.log(e);
+  process.exit(1);
+});
```

Comment:
This command is actually a command that was run once, to reinitialize the checkins of all users, and will (hopefully) never run again. I still want to keep it as a command in case it needs to run again for some reason, but basically, I don't really care if we have to revert this change (as it's just moving code from one place to another, ie moving it from the ListUserController to a command file)